how-to-guides/migration-from-5x.md

```diff
@@ -31,3 +32,15 @@ To use TinyMCE 6 from the {{site.cloudname}}, include this script in your HTML p
 ```
 
 For information on using the latest development and testing builds, see: [Cloud deployment guide - Specify editor & plugin versions]({{site.baseurl}}/cloud-deployment-guide/editor-plugin-version/).
+
+## Plugin Changes
+
+### Tabfocus plugin removed
+
+The Tabfocus plugin has been removed in {{site.productname}} 6.0. This includes the `tabfocus_elements` option.
+
+Instead, tab order should be configured by specifying the [`tabindex` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) on the target element for the editor, or the equivalent if using a framework. This allows the tab order to be handled by the browser. In inline mode, this works as in {{site.productname}} 5.x. In iframe (classic editor) mode, {{site.productname}} now copies the `tabindex` attribute from the target element to the editor's iframe, to allow this to work correctly.
```

Comment:
This migration / changelog section has more information than the actual docs that we're linking to. Should we move the bulk of this over to the actual section, and then just link to it from here?

If we do leave more content here, can we fix the "or equivalent if using a framework" here as well?

Reply:
Will prob remove the integration part. I'll check on the info. Iirc this has more because it's comparing 5 and 6 though. Or at least, that's what I intended...

Reply:
I don't think I copied any real info across, but I did add an example.